content/docs/command-reference/remote/modify.md

```diff
@@ -122,12 +135,6 @@ these settings, you could use the following options:
   $ dvc remote modify myremote secret_access_key my-secret_access_key
   ```
 
-- `url` - remote location URL:
```

Comment:
I think it worth keeping them, just for another piece of explicit information in the right context that might help to figure out what's going on

Reply:
But we have a section called **Available parameters for all remotes** and `url` is one of them. Should I add all the examples in that section? Currently I only added a few types. Or if you're sure I'll roll back the change... But that whole section only exists for `verify` right now. Seems a little pointless (i.e. I could also copy `verify` to all the type sections.

Reply:
I would expect a lot of people to start reading right from the section and skip the description. Let's not optimize it for the sake of optimization only.

Also, example (multiple dvc remote add commands) looks scary :) - it's not cleat that this is a variation of a single command, or is it something I would have do to even start with remotes.

Reply:
Optimizing wasn't even my main goal here. We currently explain `url` for each type in remote add, but in [remote modify](https://dvc.org/doc/command-reference/remote/modify#available-parameters-per-storage-type) we only mention it in 5 of them. I noticed this inconsistency and wanted to complete the info., but decided to put them all in the "for all remotes" section to avoid bloating the doc (that's the optimization part) and since it makes sense — what is that section for otherwise, just for `verify`?

> I would expect a lot of people to start reading right from the section

I personally would also check the "for all remotes" section along with the type I'm interested on.

But anyway: for now I've put URL both in the "all remotes" (general desc. and local remote example) and in every "per storage type" (specific examples except local, which doesn't have a section here). WDYT?

Reply:
sounds good!